src/asciidoc/docs/publish/plugins.adoc

```diff
@@ -0,0 +1,45 @@
+= Using Koji plugins
+:page-slug: plugins
+:page-description: Plugins
+
+A plugin is software that can be added to a Koji after publication to provide additional functionality.
+Basically, it allows for the injection of dynamic code.
+Plugins can only be added by the creator of a Koji.
+They will not be passed down to remixes.
+
+Generally, a creator customizes a plugin for a particular Koji by specifying values for one or more fields.
+For example, the *Custom Boilerplate* plugin adds a popup requiring a user to acknowledge a licensing agreement before interacting with the Koji.
+The creator specifies the body text and the question text for the popup.
+
+== Advantages of plugins
+
+In addition to allowing the developer to easily add functionality to a Koji, plugins offer a few other advantages.
+
+* Search engines and social crawlers often do not render JavaScript when indexing pages.
+For instance, if you're using React, a bot may only see the root div and an "enable JavaScript" message.
+A plugin can overcome this limitation by injecting keywords into a webpage's header.
+Because Koji plugins are computed server-side, they are available to all bots without any additional 
+configuration or third-party services.
+
+* A plugin can inject a new file at runtime, simplifying the creation of things like web app manifests.
+The developer just provides configurable values for the plugin and the injection intelligently wraps them with the code/syntax that makes sense.
```

Comment:
We might need to reframe this sentence, since "the developer" here is always someone internal to Koji. So, we might focus on how the plugin works instead of who does what. E.g., "The plugin provides configurable values, and the injection wraps them with the relevant code."